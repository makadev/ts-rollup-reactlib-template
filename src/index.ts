// Barrel File (re-export public interface)
export * from "./components/Button";
