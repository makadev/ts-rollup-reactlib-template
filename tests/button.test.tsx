import * as renderer from "react-test-renderer";
import { Button } from "../src";

it("Button renders correctly", () => {
    const props = {
        onClick: () => {
            return;
        },
        children: "test",
    };
    const tree = renderer.create(<Button {...props} />).toJSON();
    expect(tree).toMatchSnapshot();
});
