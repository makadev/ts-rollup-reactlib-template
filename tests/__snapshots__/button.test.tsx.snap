// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Button renders correctly 1`] = `
<button
  className="btn btn--secondary btn--medium"
  onClick={[Function]}
  type="button"
>
  test
</button>
`;
